docs/rst/tutorials/dashboard_inference.rst

```diff
@@ -0,0 +1,150 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _tutorials_dashboard_inference:
+
+####################################
+Make predictions with the AML Dashboard
+####################################
+
+Background
+==========
+
+The :term:`AML` Dashboard is a web-based tool that allows users to interact with the |aml| framework.
+
+This tutorial showcases the process of making predictions based on an |aml| model using the *AML Dashboard*.
+
+.. figure:: /rst/figures/tutorials/inference.png 
+    :align: center
+    :width: 100%
+
+Prerequisites
+=============
+
+Ensure you have installed the *AML Dashboard* using one of the following methods:
+
+- :ref:`Linux Installation <amldashboard_linux>`
+- :ref:`Docker Image Installation <amldashboard_docker>`
+
+For more information, check the :ref:`AML Dashboard Interfaces <amldashboard_interfaces_evaluation>` and :ref:`AML Dashboard Usage <amldashboard_usage_evaluation>` sections.
+
+Running the demo
+================
+
+To run the necessary components for evaluating a model and making predictions using the |aml| Dashboard, follow these steps:
+
+Start the backend server
+------------------------
+
+1. Navigate to the ``backend`` directory.
+
+.. code-block:: bash
+
+   cd backend
+
+2. Load the |amlip| environment.
+
+.. code-block:: bash
+
+   source /AML-IP/install/setup.bash
+
+3. Start the server:
+
+.. code-block:: bash
+
+   python3 server.py
+
+Start the Inference Node
+------------------------
+
+.. warning::
+
+    Before starting the inference node, ensure that there is at least one model and traing set available in the download directory.
+
+1. Load the |amlip| environment.
+
+.. code-block:: bash
+
+   source /AML-IP/install/setup.bash
+
+2. Navigate to the ``backend`` directory.
+
+.. code-block:: bash
+
+   cd backend
+
+3. Start one or more inference nodes:
+
+.. code-block:: bash
+
+   python3 inference.py
+
+.. check if there can be more inference nodes run parallely. 
+Each inference node will wait for job assignments and will collectively distribute the workload when multiple nodes are running. 
```

Comment:
Currently, if multiple inference nodes are run in parallel, data will only be received by one node, and inference will be performed by that single node. Although supporting parallel inference nodes can be implemented in the future, for now, I will remove the option to run multiple nodes in parallel. This will be updated once the capability is available.